src/content/docs/en/guides/images.mdx

```diff
@@ -664,13 +674,23 @@ export default defineConfig({
 });
 ```
 
-:::note
-When using a [strict package manager](https://pnpm.io/pnpm-vs-npm#npms-flat-tree) like `pnpm`, you may need to manually install Sharp into your project even though it is an Astro dependency:
+### Configure no-op passthrough service
 
-```bash
-pnpm install sharp
+If your [adapter for `server` or `hybrid` mode](https://astro.build/integrations/?search=&categories%5B%5D=adapters) does not support Astro's built-in Squoosh and Sharp image optimization (e.g. Deno, Cloudflare), you can configure a no-op image service to allow you to use the `<Image />` and `<Picture />` components. Note that Astro does not perform any image transformation and processing in these environments. However, you can still enjoy the other benefits of using `astro:assets`, including no Cumulative Layout Shift (CLS), the enforced `alt` attribute, and a consistent authoring experience.
+
+Configure the no-op image service to avoid both Squoosh and Sharp image processing:
+
+```js title="astro.config.mjs" ins={4-8}
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  image: {
+    service: {
+      entrypoint: 'astro/assets/services/noop'
+    }
+  }
+});
```

Comment:
```suggestion
```js title="astro.config.mjs" ins={4-8}
import { defineConfig, passthroughImageService } from 'astro/config';

export default defineConfig({
  image: {
    service: passthroughImageService()
  }
});
```

We have a handy function! I did not adjust the highlight because I'm unfamiliar with the syntax

Reply:
Ah, I had this one in my notes, and was going to ask you which one I should use, but thought it was safer just to recopy the old one. Perfect!